CenterForOpenScience/osf.io
# Adding branched from node. [ENG-1898]

## Purpose

To make analytics easier, a database column `branched_from_node` needs to be added.

## Changes

Adding field in model, adding code for population at registration time. Adding migration and management command to populate new field.

## QA Notes

Workflows impacted: creating registrations both ways

Does this change require a data migration? Yes

What is the level of risk?

Any permissions code touched?

Is this an additive or subtractive change, other?

How will QA verify? (UI, API, admin app or AdminAdminApp?) DevTest, Courtney directly queries DB

How will this impact performance?

## Documentation

N/A

## Side Effects

N/A

## Ticket

https://openscience.atlassian.net/browse/ENG-1898
